BrainPortal/app/helpers/rich_ui_helper.rb

```diff
@@ -336,6 +336,40 @@ def button_with_dropdown_menu(title, options={}, &block)
     return html
   end
 
+  # Create copy to clipboard button
+  # On click on this button a text `txt` is copied to the clipboard.
+  # Default label for the button is text 'copy', followed by svg icon for copying
+  # In the case of successful copying a tooltip like message 'Copied!' rather than an alert
+  #
+  # Options:
+  # [:label] a button label. Defaults to 'copy' followed by a copy icon
+  # [:message] feedback message. Default to 'Copied!'.  #
+  # All other options are treated as HTML attributes of the button
+  def copy_to_clipboard_button(txt, options={})
+    ok_msg   = options.delete(:message)
+    ok_msg ||= 'Copied!'
+    label    = options.delete(:label)
+    label  ||= 'copy' +
```

Comment:
I'd like the caller to be able to change the text label and still get the icon. Right now if I change the label, the icon disappear.

Also the default label should be the full "Copy to clipboard", because just the word "copy" is not clear enough.